wallet/service/v2/connections/store/v1/file_store.go

```diff
@@ -170,8 +170,12 @@ func (s *FileStore) Close() {
 	}
 }
 
-func (s *FileStore) readTokensFile() (tokensFile, error) {
-	tokens := tokensFile{}
+func (s *FileStore) readTokensFile() (tokens tokensFile, rerr error) {
+	defer func() {
+		if r := recover(); r != nil {
+			tokens, rerr = tokensFile{}, fmt.Errorf("unexpected fs issues reading tokens file: %s", r)
```

Comment:
those errors can propagate to regular users. Maybe we could make them a little bit more user friendly, knowing it will never be perfectly accessible.

What about:
`an system error occurred while reading the tokens file: %w`